### PR TITLE
Fix fallback logic for image and video element accessibility and title fields

### LINKED ISSUE #1128: Unable to clear alt text, video and image elements
## Bug Description
I am unable to clear the alt / title accessibility fields on the video element. When I empty the field, either by using the delete key or cross, it refills with default value again. 

## Expected Behaviour
User should be able to clear field. 

## Steps to Reproduce

See video. 

## Screenshots

https://youtu.be/DEq9MiOyjI4

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by diegovar**
This fixes #1128 (the issue where the alt and title tags cannot be set to an empty string in the inspector).

I believe this is a stop-gap fix. A more comprehensive fix for the meaning of null/emptyString/undefined for properties in general needs to be thought out.

### CODE REVIEW

**Comment by spacedmonkey** [comment]
Does this fix #1128 ?

File: assets/src/edit-story/components/panels/videoAccessibility.js [hunk 1]
```diff
@@ -41,9 +41,21 @@ function VideoAccessibilityPanel({ selectedElements, pushUpdate }) {
     'resource',
     DEFAULT_RESOURCE
   );
-  const poster = getCommonValue(selectedElements, 'poster') || resource.poster;
-  const title = getCommonValue(selectedElements, 'title') || resource.title;
-  const alt = getCommonValue(selectedElements, 'alt') || resource.alt;
+
+  const _defaultForUndefined = (s, d) => (s === undefined ? d : s);
+
+  const poster = _defaultForUndefined(
```
Comment (spacedmonkey) [inline assets/src/edit-story/components/panels/videoAccessibility.js]: Isn't the third param on `getCommonValue` already default? Why not just reuse that?

File: assets/src/edit-story/components/panels/videoAccessibility.js [hunk 2]
```diff
@@ -41,9 +41,21 @@ function VideoAccessibilityPanel({ selectedElements, pushUpdate }) {
     'resource',
     DEFAULT_RESOURCE
   );
-  const poster = getCommonValue(selectedElements, 'poster') || resource.poster;
-  const title = getCommonValue(selectedElements, 'title') || resource.title;
-  const alt = getCommonValue(selectedElements, 'alt') || resource.alt;
+
+  const _defaultForUndefined = (s, d) => (s === undefined ? d : s);
```
Comment (dvoytenko) [inline assets/src/edit-story/components/panels/videoAccessibility.js]: No need for underscore. Also, if it's completely pure - let's move out of the function body on the top level.

File: assets/src/edit-story/components/panels/videoAccessibility.js (hunk 1 above)
Comment (dvoytenko) [reply assets/src/edit-story/components/panels/videoAccessibility.js]: Yep. Looks like `def` arg works the same way as `undefined`.

File: assets/src/edit-story/elements/video/output.js [hunk 3]
```diff
@@ -27,12 +27,14 @@ function VideoOutput({ element, box }) {
     type: resource.mimeType,
     src: resource.src,
   };
+
+  const _defaultForUndefined = (s, d) => (s === undefined ? d : s);
```
Comment (dvoytenko) [inline assets/src/edit-story/elements/video/output.js]: Ditto: naming and location.

**Review by dvoytenko — CHANGES_REQUESTED** [review]

File: assets/src/edit-story/components/panels/videoAccessibility.js (hunk 2 above)
Comment (diegovar) [reply assets/src/edit-story/components/panels/videoAccessibility.js]: I've removed the method as I can just use the 3rd param of getCommonValue.

**Comment by diegovar** [comment]
Sorry for the confusion, I've closed https://github.com/google/web-stories-wp/pull/1341 and rolled it into this PR.

**Review by dvoytenko — APPROVED** [review]

**Pull request merged**